Content.Server/Weapon/Ranged/GunSystem.Battery.cs

```diff
@@ -30,11 +27,8 @@ private void OnBatteryMapInit(EntityUid uid, BatteryBarrelComponent component, M
 
     private void OnBatteryGetState(EntityUid uid, BatteryBarrelComponent component, ref ComponentGetState args)
     {
-        (int, int)? count = (component.ShotsLeft, component.Capacity);
-
-        args.State = new BatteryBarrelComponentState(
-            component.FireRateSelector,
-            count);
+        // currently unused, but maybe `FireRateSelector` will be used eventually?
+        args.State = new BatteryBarrelComponentState(component.FireRateSelector);
```

Comment:
I could just remove the networking altogether if this is not likely to be used.

Reply:
Client will need to know about fire rate for prediction

Reply:
I'll just remove for now then.